### PR TITLE
Create a Participant Page to Show Participants and Their Filters (Re-Merge)

### PR DESCRIPTION
**Pull request by aivey65**
### Create a Participant Page
_(This pull request is a remerge of #15)_
Linked Issue: #5 
**Changes**
- Create an HTML file to show participants.
- In participants.js, create HTML elements with sample data to show how participants in the session will be displayed on the UI.

**Attached**
- Image of existing UI with sample data and basic styling
_(The styling in the image is just to show how different elements could be styled. Styling is not a part of this pull request.)_
- Image of HTML code after it gets updated by participants.js

Screenshot 2020-07-09 at 1 30 51 PM

Screenshot 2020-07-09 at 1 43 48 PM

### CODE REVIEW

**Comment by ChisomOkwor** [comment]
Looks good Asha

File: webapp/javascript/participants.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  allParticipants.forEach(participant => {
+    var newDiv = document.createElement('div');
+    newDiv.classList.add('participant');
+
+    var name = document.createElement('p');
+    name.classList.add('participant-name');
+    name.appendChild(document.createTextNode(participant.name));
+    newDiv.appendChild(name);
+
+    var location = document.createElement('p');
```
[the remark this answers is not in the document]
Comment (vedantroy) [reply webapp/javascript/participants.js:33]: Oh, whoops too late! (Didn't realize you need to submit reviews). We can fix this later :)

[the remark this answers is not in the document]
Comment (vedantroy) [reply webapp/javascript/participants.js:33]: Would recommend taking a look at this: https://google.github.io/styleguide/jsguide.html#features-local-variable-declarations

[the remark this answers is not in the document]
Comment (aivey65) [reply webapp/javascript/participants.js:33]: I will fix this with my next changes. Thanks Ved!

**Pull request merged**